### ISSUE TITLE
proselint.DateCase: false positive

### ISSUE THREAD

**Issue by acalvo** [errata-ai/proselint#3]
The text (address) "1600 Amphitheatre Pkwy, Mountain View, CA 94043" is reporting the following error:

```
 1:3  error  With lowercase letters, the     proselint.DateCase 
             periods are standard.                             
```

It's due to `1600 Amphitheatre`, that matches `\d{1,2} ?[ap]m`. That's not intended, is it? :)

**Comment by jdkato** [comment]
I fixed this by adding a word boundary to the right-hand side. I'll put out a new release soon.
